cluster/libvirt-coreos/network_kubernetes_pods.xml

```diff
@@ -1,6 +1,6 @@
 <network>
   <name>kubernetes_pods</name>
   <bridge name='virbr_kub_pods' stp='off' delay='0'/>
-  <ip address='10.10.0.1' netmask='255.255.0.0'>
+  <ip address='10.10.0.100' netmask='255.255.0.0'>
```

Comment:
@mikedanese Both 10.10.*.* is ok. The conflict is master's IP is 10.10.0.1, and virbr_kub_pods's IP is 10.10.0.1 too. This cause can't PING the pod IP on master. Because the PING package is done on virbr_kub_pods and can't pass the master vm's veth.